tests/Operation/AggregateFunctionalTest.php

```diff
@@ -221,9 +221,7 @@ function () use ($pipeline, $options) {
                 $this->assertCount(1, $results);
                 $result = current($results);
 
-                if ($this->isShardedCluster()) {
-                    $this->assertObjectHasAttribute('shards', $result);
-
+                if (isset($result->shards)) {
```

Comment:
This was necessary as the output of `explain` on a cluster with a single shard does not contain the `shards` key but rather looks like the result on a standalone.